client/coral-admin/src/routes/Moderation/components/UserDetail.js

```diff
@@ -90,8 +98,14 @@ export default class UserDetail extends React.Component {
     return (
       <Drawer handleClickOutside={hideUserDetail}>
         <h3>{user.username}</h3>
-        <Button className={styles.copyButton} onClick={this.copyPermalink}>Copy</Button>
         {profile && <input className={styles.profileEmail} readOnly type="text" ref={(ref) => this.profile = ref} value={profile} />}
+
+        <Copy onCopy={() => this.showCopied()} text={profile} className={styles.profileEmail}>
```

Comment:
className property is passed but `Copy` button doesn't seem to support `className`.